src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml

```diff
@@ -1,4 +1,8 @@
 @{
+    if (Model.Type == null)
```

Comment:
IMHO I think we know what's the reason that make it `null` instead of `return`

Reply:
I don't think I understand the ask.

Reply:
Sorry I miss "need to" ;)

Reply:
I don't know the real reason why it becomes null. but I have encounter the null reference exception couple times and I think a null check is a valid catch to prevent the exception.

Reply:
Please steps to reproduce, otherwise we cann't accept this as a proper solution

Reply:
Hmm, normally the model is built from a `NotifyEntry` and `Type` is an **enum field**.

It's like a "Message" shape whose dynamic model would not be built from a `NotifyEntry`.

Reply:
Not sure why it happens for me every now and then. Next time it happen, I’ll try to see if I can come up with steps to reproduce